build/IPDK_Container/scripts/set_hugepages.sh

```diff
@@ -4,8 +4,14 @@
 
 #...Setting Hugepages...#
 mkdir -p /mnt/huge
-mount -t hugetlbfs nodev /mnt/huge
-echo -e "nodev /mnt/huge hugetlbfs\n" >> /etc/fstab
+if [ "$(mount | grep hugetlbfs)" == "" ]
+then
+	mount -t hugetlbfs nodev /mnt/huge
+fi
+if [ "$(grep huge < /etc/fstab)" == "" ]
+then
+	echo -e "nodev /mnt/huge hugetlbfs\n" >> /etc/fstab
```

Comment:
Running mount by hand doesn't automatically update /etc/fstab.

Reply:
No, but why do you need to add it to fstab (that is only used at startup) if the check if the mount is there is always done at run of the script? (and if its not there it will be added...)

Reply:
Ah, since this is running in a persistent Vagrant VM, the check is there because if you reload the Vagrant VM, it will exist in /etc/fstab and be automatically mounted, and both conditions here will work correctly.